dle/search/static/search/es_search.js

```diff
@@ -36,6 +37,10 @@
   
       search.addWidgets([
           instantsearch.widgets.searchBox({
+              queryHook(query, search) {
+                  globalSearchTerm = query;
```

Comment:
A hook on each search so we can grab the search term to use in the templates section.